pkgs/development/python-modules/pyisbn/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, hypothesis, lib, pytest, pytest-randomly
+, pytest-cov }:
+
+buildPythonPackage rec {
+  pname = "pyisbn";
+  version = "1.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06fm9rn31cb4b61hzy63cnwfjpppgyy517k8a04gzcv9g60n7xbh";
+  };
+
+  checkInputs = [ pytest-randomly pytest-cov pytest hypothesis ];
```

Comment:
We don't care about coverage. Remove `pytest-cov` please and adjust `addopts` in `setup.cfg`.

```
  postPatch = ''
    substituteInPlace setup.cfg \
      --replace "--cov pyisbn --cov-report term-missing --no-cov-on-fail" ""
  '';
```

Use `pytestCheckHook` instead of `pytest` if possible